public/locales/ja-JP/translations.json

```diff
@@ -1,34 +1,37 @@
 {
+  "action": "アクション",
   "assets": "資産",
   "assets.nft_tab_title": "非代替性トークン",
   "assets.issued_tab_title": "発行済み通貨",
   "assets.no_issued_message": "トークンが見つかりません。",
-  "assets.no_nfts_message": "NFTが見つかりません",
-  "network_name_testnet": "testnetデータ",
-  "network_name_mainnet": "liveデータ",
-  "network_name_devnet": "devnetデータ",
-  "network_name_amm": "AMM-Devnetデータ",
-  "network_name_custom": "カスタムネットワークデータ",
+  "assets.no_nfts_message": "NFTが見つかりません。",
+  "network": "ネットワーク",
+  "network_name": "不明なネットワーク",
+  "network_name_testnet": "Testnet",
+  "network_name_mainnet": "Mainnet",
+  "network_name_devnet": "Devnet",
+  "network_name_amm": "AMM-Devnet",
+  "network_name_custom": "カスタムネットワーク",
```

Comment:
I updated the english to only have the word "Custom" instead of "Custom Network".

I don't know how that will change the translation.

Reply:
The Japanese-speaking XRPL community does not currently use the term "Custom Network", so honestly, anything other than the Live/Dev/Test network is fine as long as it is understood.

I'll just use "Custom" as it is.